recipes/requests-kerberos/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "requests-kerberos" %}
+{% set version = "0.12.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "9d21f15241c53c2ad47e813138b9aee4b9acdd04b82048c4388ade15f40a52fd"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
```

Comment:
Please add `'noarch: python`.

Reply:
@npavlovikj, my understanding is that `noarch: python` is not appropriate for packages that have platform-specific runtime requirements, as this does.